sdk/attestation/azure-security-attestation/test/ut/policygetset_test.cpp

```diff
@@ -252,18 +252,18 @@ namespace Azure { namespace Security { namespace Attestation { namespace Test {
 
     /** @brief Tests for policy modification using an unsecured JWS.
      *
-     * Forwards to the `SetPolicyTest` and `ResetPolicyTest` with a non-present AttestationSigingKey
+     * Forwards to the `SetPolicyTest` and `ResetAttestationPolicyTest` with a non-present AttestationSigingKey
      * parameter.
      */
     void ModifyPolicyUnsecuredTest()
     {
       SetPolicyTest();
-      ResetPolicyTest();
+      ResetAttestationPolicyTest();
```

Comment:
The test changes don't seem necessary (and might be a side effect of find/replace).

But if we are modifying `ResetPolicyTest`, we should also consider updating `SetPolicyTest` to match.

Reply:
Agreed, I had reverted the test changes but missed this file.  Looking at it, there are a couple comments that make sense to keep.  I've undone the changes to tests while leaving the comments referring to the SDK methods